src/AndorSIF.jl

```diff
@@ -184,20 +190,20 @@ function FileIO.load(fs::Stream{format"AndorSIF"})
     for i = 1:frames
         _ = readline(io)
     end
-    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
 
-    pixels = read(io, Float32, width, height, frames)
-    prop = Compat.@compat Dict(
-        "colorspace" => "Gray",
-        "spatialorder" => ["y", "x"],
-        "ixon" => ixon,
-        "suppress" => Set{Any}(("ixon",)),
-        "pixelspacing" => [1, 1]
+    # two additional lines?
+    _ = readline(io)
+    _ = readline(io)
+
+    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
+    numbytes = 4 * frames * width * height # number of bytes to read
+    pixelbytes = read(io, numbytes)
+    pixelarray = reinterpret(Gray{Float32}, pixelbytes)
+    pixelmatrix = reshape(pixelarray, (height, width)) |> collect
```

Comment:
I'm not sure if it is a good idea to collect the reshaped array. But returning the `Reshaped` type looked a bit messy.

Reply:
You could also do this:
```julia
pixelmatrix = Matrix{Gray{Float32}}(undef, height, width)
read!(io, pixelmatrix)
```
and then it will immediately be in the shape and element type you want.

Reply:
Nice trick! That made me realize that there currently was no support for multiple frames. Returning a 3D array would be a solution. I don't know if that's the best, though.

Reply:
If it's continuous on disk, that would be best. Even better would be to use `Mmap.mmap`: https://docs.julialang.org/en/latest/stdlib/Mmap/, since it would allow you to "load" a terabyte-sized file without saturating your memory. You can see examples in https://github.com/JuliaIO/NRRD.jl/blob/9202ea1d01c69413104e2e7328f2cbaca06f5ee2/src/NRRD.jl#L242-L243.